### DISCUSSION TITLE
linux/wiznote

### DISCUSSION THREAD

**Discussion by giscus** [LaoYutang/blogGiscus#13] [Announcements]
# linux/wiznote

本文将推荐博主自用的私有笔记服务为知笔记WizNote，采用Docker部署，镜像为博主根据官方镜像修改，解除了某些限制。

https://blog.laoyutang.cn/linux/wiznote.html

**Comment by yinhuihe** [comment]
大佬  我用26版本的docker mysql起不来  有没有时间研究一下

**Comment by lovebout** [comment]
大佬，能做一个arm的镜像吗

**Comment by umbrellax** [comment]
想咨询下 自己改造了crx插件 把转发到了wiz.cn指向了自己私有化部署，目前测试登录 获取分类都没问题。就是剪辑保存的时候 我通过抓包显示响应Not Found 。https://xxxxx/ks/gather?clientType=webclip_chrome&clientVersion=4.0.10&apiVersion=10&lang=zh-CN  响应为“Not Found” 请问如何解决
